### DISCUSSION TITLE
Is the speech2text option avaialble in chatgpt-ui?

### DISCUSSION THREAD

**Discussion by sumeetdas** [mckaywrigley/chatbot-ui#1417] [Help]
I saw a couple of issues ([this](https://github.com/mckaywrigley/chatbot-ui/issues/21) and [this](https://github.com/mckaywrigley/chatbot-ui/pull/855)) which requested for speech2text feature using Whisper. These issues were closed as 'Completed' but I can't find this option anywhere in the app. Is the speech2text option available in chatgpt-ui? I'm interested in calling a locally deployed Whisper service in chatgpt-ui to convert my voice to chat input.

**Comment by secondtruth** [comment]
The issues you mentioned are from the legacy version. They were closed in favor of v2, where speech to text is sadly not yet available either.

**Comment by jralduaveuthey** [comment]
any branch or similar where one can get an implementation of whisper for the new version of chatbotui?
